scalableminds/webknossos
# Moved routing for /datasets to frontend

### Steps to test:
- Make sure the routes `/dataset/<name>/view` and `/dataset/<name>/edit` still work:
- Go to Dashboard.Advanced Datasets View:
  - Click "edit" --> routing should work. Refresh page --> routing should work
  - Click "view" --> routing should work. Refresh page --> routing should work

### Issues:
- related to #2319 

## Attention. Change base branch before merging.
------
